client/src/components/SharedComponents/Signup/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import './signup.css'
+
+class Signup extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    confirm_password: '',
+    displayBio: false
+  }
+
+  onChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  ShowDisplayBio = () => {
```

Comment:
we can replace the two bio functions with one.
If this is confusing then don't do it. It's just a suggestion.
```suggestion
  toggleDisplayBio = () => {
      this.setState({ displayBio: !this.state.displayBio }) // Sets state to ```true``` if value is ```false``` and vice versa
```